docs/handbook/01_intro.md

```diff
@@ -0,0 +1,19 @@
+This entails high-level support for end users in realizing their _applications_ {cite:p}`quetschlichAutomatedFrameworkRealizing2023,quetschlichHybridClassicalQuantum2023,quetschlichPredictingGoodQuantum2023,quetschlichUtilizingResourceEstimation2024,quetschlichApplicationawareQuantumCircuit2024, quetschlichMQTPredictorAutomatic2023` as well as efficient methods for the _classical simulation_ {cite:p}`zulehnerAdvancedSimulationQuantum2019,zulehnerMatrixVectorVsMatrixMatrix2019,hillmichJustRealThing2020,hillmichAccurateNeededEfficient2020,hillmichApproximatingDecisionDiagrams2022,grurlConsideringDecoherenceErrors2020,grurlStochasticQuantumCircuit2021,grurlNoiseawareQuantumCircuit2023,burgholzerHybridSchrodingerFeynmanSimulation2021,sanderHamiltonianSimulationDecision2023,matoMixeddimensionalQuantumCircuit2023,hillmichConcurrencyDDbasedQuantum2020,burgholzerExploitingArbitraryPaths2022,burgholzerSimulationPathsQuantum2022,burgholzerEfficientConstructionFunctional2021`, _compilation_ {cite:p}`zulehnerEfficientMethodologyMapping2019,hillmichExploitingQuantumTeleportation2021,zulehnerCompilingSUQuantum2019,willeMappingQuantumCircuits2019,burgholzerLimitingSearchSpace2022,pehamDepthoptimalSynthesisClifford2023, schneiderSATEncodingOptimal2023,pehamOptimalSubarchitecturesQuantum2023,quetschlichMQTPredictorAutomatic2023,quetschlichCompilerOptimizationQuantum2023,quetschlichReducingCompilationTime2023,schoenbergerUsingBooleanSatisfiability2024,schoenbergerCyclebasedShuttlingTrappedIon2024,schoenbergerShuttlingScalableTrappedIon2024,schmidHybridCircuitMapping2024,schmidComputationalCapabilitiesCompiler2024,matoAdaptiveCompilationMultilevel2022,matoCompilationEntanglingGates2023,matoCompressionQubitCircuits2023,matoMixeddimensionalQuditState2024`, and _verification_ {cite:p}`burgholzerAdvancedEquivalenceChecking2021,burgholzerImprovedDDbasedEquivalence2020, burgholzerPowerSimulationEquivalence2020,burgholzerRandomStimuliGeneration2021, burgholzerVerifyingResultsIBM2020,pehamEquivalenceCheckingParameterized2023,pehamEquivalenceCheckingQuantum2022,pehamEquivalenceCheckingParadigms2022,willeVerificationQuantumCircuits2022` of quantum circuits.
+Reaching towards the hardware level, we also consider tools for _quantum error correction_ {cite:p}`berentDecodingQuantumColor2023,berentSoftwareToolsDecoding2023,strikisQuantumLowdensityParitycheck2023` and _physical design_ {cite:p}`kunasaikaranFrameworkDesignRealization2024`.
+In all these tools, we try to utilize _data structures_ (such as decision diagrams) and _core methods_ (such as reasoning engines) to facilitate the efficient handling of quantum computations.
+The proposed solutions demonstrate how utilizing design automation expertise can lead to improved efficiency, scalability, and reliability.
+They illustrate the immense benefits of leveraging expertise in classical circuit and system design rather than starting from scratch.
```

Comment:
To fit the sentences more smoothly maybe put a "In particular, they illustrate" or similar?

Reply:
Would you mind just adding things like that as a literal suggestion on GitHub? That way it is just the click of a button to incorporate 😌

Reply:
Nvm. Done in https://github.com/cda-tum/mqt/pull/10/commits/e899ef94b19899ccfae454ed608493d501de63cd 👍🏼

Reply:
oh sorry missed that, thanks a lot for the effort!